src/modules/views/OnlyOffice/OnlyOfficeProvider.jsx

```diff
@@ -82,11 +84,12 @@ const OnlyOfficeProvider = ({
   useEffect(() => {
     if (
       searchParam.get('fromCreate') === 'true' ||
-      searchParam.get('fromEdit') === 'true'
+      searchParam.get('fromEdit') === 'true' ||
+      hasWriteAccess(fileId, driveId)
```

Comment:
why keeping searchParam and "edit button"? Now we don't want to use the edit button anymore, If a user has a write access it doesn't make sens to keep it. And if a user doesn't have write access, it probably doesn't make sens either 🤔 I think here we can simplify things and rely only on hasWriteAccess, isn't it?